roles/setup/tasks/setup-container.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Enable linger for {{ receptor_user }}
+  become: true
+  ansible.builtin.command:
+    cmd: "loginctl enable-linger {{ receptor_user }}"
+  changed_when: false
+
+- name: Get UID of {{ receptor_user }}
+  ansible.builtin.command:
+    cmd: "id -u {{ receptor_user }}"
+  register: receptor_uid
+  changed_when: false
+
+- name: Setup container
+  become: true
+  become_user: "{{ receptor_user }}"
+  environment:
+    XDG_RUNTIME_DIR: "/run/user/{{ receptor_uid.stdout }}"
+  vars:
+    receptor_uid_directory: "/run/user/{{ receptor_uid.stdout }}"
+  block:
+    - name: Enable podman socket
+      ansible.builtin.systemd:
+        name: podman.socket
+        state: started
+        enabled: true
+        scope: user
+
+    - name: Pull the latest release of receptor from Quay
+      containers.podman.podman_image:
+        name: "{{ receptor_image_name }}"
+        tag: "{{ receptor_image_tag }}"
+        state: present
+
+    - name: Run receptor image
+      containers.podman.podman_container:
+        name: "{{ receptor_service_name }}"
+        image: "{{ receptor_image_name }}:{{ receptor_image_tag }}"
+        state: created
```

Comment:
Ill change this to `started`, and configure the handler to restart the systemd service.

This flow of `creating` and starting the container with systemd, we used in self-managed ansible, so i used it here as well initially.

Reply:
I think it may be better to keep the current flow of `created/started` as of now.

Setting it to restarted causes systemd to continually hit issues when it restarts, even though it appears to be configured exactly as expected - 
```
fatal: [remote-execution]: FAILED! => {"changed": false, "msg": "Unable to start service receptor: Job for receptor.service failed because the service did not take the steps required by its unit configuration.\nSee \"systemctl --user status receptor.service\" and \"journalctl --user -xeu receptor.service\" for details.\n"}
```

I am seeing this podman issue pop up from time to time as well, which breaks the idempotency - https://github.com/containers/podman/issues/7889
To get past this, I need to install `buildah` and manually remove the build container.